components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -357,6 +357,10 @@ open class Nimbus(
         recordExposure(featureId)
     }
 
+    override fun recordEvent(eventId: String) {
+        nimbusClient.recordEvent(eventId)
```

Comment:
This does potentially have some IO writing going on, I think that launching this on the dbScope may be the right thing to do here so we aren't blocking the calling thread.

Reply:
The only reason `recordExposure` doesn't do this above, is because Glean already manages the async bit for us.